gott.go

```diff
@@ -215,15 +200,10 @@ func main() {
 		config = <-cacheChan
 	} else {
 		config = parseToml(tomlFiles, tomlText)
-		realizeConfig(config, config, []string{})
-	}
-
-	for _, p := range promotions {
-		config = config.Promote(strings.Split(p, "."))
-	}
-
-	if narrow != "" {
-		config = config.Narrow(strings.Split(narrow, "."))
```

Comment:
and removing these pieces also unfortunately removes some features 🙁, not sure how much they're needed for your end goal

Reply:
I will want to keep this feature -- example usage [here](https://github.com/neeasade/dotfiles/blob/40f05d4f0caa62e1f2acc31f892a211adf7e5f3e/bin/bin/ltheme#L57) and [here](https://github.com/neeasade/dotfiles/blob/40f05d4f0caa62e1f2acc31f892a211adf7e5f3e/wm/.wm/panel/tools/mkb#L9-L15)